spec/spec_helper_acceptance.rb

```diff
@@ -31,28 +31,25 @@
   # Setup Puppet Bolt
   gitlab_ip = File.read(File.expand_path('~/GITLAB_IP')).chomp
   bolt = <<-MANIFEST
-  $bolt_config = @("GITCONFIG"/L)
+  $bolt_config = @("BOLTPROJECT"/L)
   modulepath: "/etc/puppetlabs/code/modules:/etc/puppetlabs/code/environments/production/modules"
-  ssh:
-    host-key-check: false
-    user: root
-    password: root
-  | GITCONFIG
+  analytics: false
+  | BOLTPROJECT
 
   package { 'puppet-bolt':
     ensure => installed,
   }
 
-  file { [ '/root/.puppetlabs', '/root/.puppetlabs/bolt']:
+  file { [ '/root/.puppetlabs', '/root/.puppetlabs/bolt', '/root/.puppetlabs/etc', '/root/.puppetlabs/etc/bolt']:
     ensure => directory,
   }
 
-  file { '/root/.puppetlabs/bolt/analytics.yaml':
+  # Needs to existing to not trigger a warning sign...
+  file { '/root/.puppetlabs/etc/bolt/analytics.yaml':
     ensure  => file,
-    content => "disabled: true\n",
```

Comment:
Can we keep this part?

Reply:
We do :) It's moved to the `bolt-project.yaml` :)

Reply:
Ah right ...